src/libs/Notification/LocalNotification/BrowserNotifications.js

```diff
@@ -117,4 +117,15 @@ export default {
             onClick,
         });
     },
+
+    /**
+     * Create a notification to indicate that an update is available.
+     */
+    pushUpdateAvailableNotification() {
+        push({
+            title: 'Update available',
+            body: 'A new version of Expensify.cash is available for download.',
```

Comment:
Is it "available for download", or is it already downloaded?

```suggestion
            body: 'A new version of Expensify.cash is available!',
```